PointerGenerator failure with unsigned __int128[]

I see the following failure with `PointerGenerator` for the type `unsigned __int128[]` in porting OpenJ9-JDK17 to a new platform.

```
Writing generated classes to /Users/openj9/openj9-openjdk-jdk17/build/macosx-aarch64-server-release/support/gensrc/openj9.dtfj/com/ibm/j9ddr/vm29/pointer/generated
java.lang.IllegalArgumentException: Type is not a recognized array: unsigned __int128[]
        at com.ibm.j9ddr.tools.PointerGenerator.getArrayType(PointerGenerator.java:930)
        at com.ibm.j9ddr.tools.PointerGenerator.writeArrayMethod(PointerGenerator.java:843)
        at com.ibm.j9ddr.tools.PointerGenerator.generateImplementationMethods(PointerGenerator.java:589)
        at com.ibm.j9ddr.tools.PointerGenerator.generateClass(PointerGenerator.java:349)
        at com.ibm.j9ddr.tools.PointerGenerator.generateClasses(PointerGenerator.java:175)
        at com.ibm.j9ddr.tools.PointerGenerator.main(PointerGenerator.java:122)
Pointer class generation failed
make[4]: *** [/Users/openj9/openj9-openjdk-jdk17/build/macosx-aarch64-server-release/support/ddr/gensrc-pointers.done] Error 1
```

I tried adding `__int128` and `unsigned __int128` to the file runtime/ddr/excludes, but it didn't work -- I saw the same failure.

--- a/debugtools/DDR_VM/src/com/ibm/j9ddr/tools/PointerGenerator.java
+++ b/debugtools/DDR_VM/src/com/ibm/j9ddr/tools/PointerGenerator.java
@@ -840,8 +840,14 @@ public class PointerGenerator {
 
 	private void writeArrayMethod(PrintWriter writer, StructureDescriptor structure, FieldDescriptor fieldDescriptor) {
 		try {
-			String returnType = getArrayType(fieldDescriptor.getType());
-			if (returnType.equals("EnumPointer")) {
+			String fieldType = fieldDescriptor.getType();
+			String returnType = getArrayType(fieldType);
+
+			if (returnType == null) {
+				String error = String.format("Unhandled array type: %s->%s %s",
+						structure.getPointerName(), fieldDescriptor.getName(), fieldType);
+				System.out.println(error);
+			} else if (returnType.equals("EnumPointer")) {
 				writeEnumEAMethod(writer, returnType, structure, fieldDescriptor);
 			} else {
 				writeEAMethod(writer, returnType, structure, fieldDescriptor);
@@ -927,7 +933,7 @@ public class PointerGenerator {
 			}
 		}
 
-		throw new IllegalArgumentException("Type is not a recognized array: " + arrayDeclaration);
+		return null;
 	}
 
 	private void writeBoolMethod(PrintWriter writer, StructureDescriptor structure, FieldDescriptor fieldDescriptor) {
@@ -1080,7 +1086,7 @@ public class PointerGenerator {
 			writer.format("\tprivate %s %s_cache;%n", typeString, getter);
 		}
 		if (writeMethodSignature(writer, returnType, getter, fieldDescriptor, true)) {
-			String accessor = simpleTypeAccessorMap.get(type);
+			String accessor = simpleTypeAccessorMap.get(Integer.valueOf(type));
 			switch (type) {
 			case TYPE_IDATA:
 			case TYPE_UDATA:

